vtex-apps/tiktok-tbp
# Feature/brpa 714 add events trigger

#### What is the purpose of this pull request?

- Pixel events trigger added: AddToCart, ViewContent and PlaceAnOrder.

#### How to test it?

You will need the TikTok Pixel Helper extension for Chrome and a valid pixel ID set in the app's settings. You can access [this workspace](https://tiktoktbp--sandboxbrdev.myvtex.com/) and see the events being triggered by accessing pages, accessing product pages (product params are passed) and adding to cart (items params are also passed).

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/50276597/149528474-beec3a58-5f99-4956-b823-f8907a2a40a7.png)

![image](https://user-images.githubusercontent.com/50276597/149529282-57ec6fac-7a69-4f57-afca-499aeb2d1b79.png)

![image](https://user-images.githubusercontent.com/50276597/149529389-f3b7f3c7-d678-45b0-8c2f-8af0b1027a75.png)

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/n5iCWgpOnRG9cFjQUU/giphy.gif)

#### Types of changes
